### DISCUSSION TITLE
MISP Installation on RHEl 8

### DISCUSSION THREAD

**Discussion by siddarthtalupula** [MISP/MISP#8624] [Q&A]
Hello, 

I have attempted install MISP on RHEL 8 multiple times, and the installation fails. Is anyone successfully installed MISP on RHEL 8. 

![image](https://user-images.githubusercontent.com/58330347/192798865-515292a0-6282-4c0c-bd83-6833b1e10926.png)

![image](https://user-images.githubusercontent.com/58330347/192798694-9c41f5e7-b5ed-46d3-a98b-7fbe64da40d7.png)

I get repeated error: **/var/www/MISP/app/Console/cake: line 31: exec: php: not found**, and finally it tells **Installation Successfully**, but I am unable to access the service - as the service is not running, and I was not prompted creating MISP account.

Thanks in advance,
Siddarth

**Comment by lukaszs1942** [comment]
Yes i did,

After installation need to update submodules and install cakereque + set correct permissions

dnf install https://rpms.remirepo.net/enterprise/remi-release-8.rpm

sudo -u apache php composer.phar require kamisama/cake-resque:4.1.2
sudo -u apache php composer.phar config vendor-dir Vendor
sudo -u apache php composer.phar install

# submodules

git submodule init
git submodule update
